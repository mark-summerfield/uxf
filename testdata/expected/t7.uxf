uxf 1.0 TLM Config
=Files Kind Filename
{
  <General>
    {#<Map comment> 
      <saved> 2022-03-21
      <autosave> yes
      <historysize> -35
      <volume> 0.7
      <files>
        (#<Table comment> Files
          <current> </home/mark/app/rs/tlm/PlaylistsTest.tlm>
          <recent1> </home/mark/app/rs/tlm/PlaylistsTest.tlm>
          <recent2> </home/mark/data/playlists-all.tlm>
        )
    }
  <Window> {<x> 383 <y> 124 <width> 590 <height> 536 <scale> 1.1}
  <Magic> (:1F8B:)
  <Nested Map>
    {
      <Classical> [5 yes]
      <Modern Instrumental> [4 no]
      <New Acquistions> [1 no]
      <Nested List of Lists>
        [#<List comment> 
          [5 <Classical &amp; &lt;Special&gt;> yes]
          [4 <Modern Instrumental> no]
          [1 <New Acquistions> ?]
        ]
    }
}
