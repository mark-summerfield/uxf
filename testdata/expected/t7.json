{
  "UXF^custom": "TLM Config",
  "UXF^ttypes": [
    {
      "UXF^ttype": {
        "name": "Files",
        "fields": {
          "Kind": null,
          "Filename": null
        }
      }
    }
  ],
  "UXF^data": {
    "General": {
      "UXF^map": {
        "comment": "Map comment",
        "ktype": null,
        "vtype": null,
        "map": {
          "saved": {
            "UXF^date": "2022-03-21"
          },
          "autosave": true,
          "historysize": -35,
          "volume": 0.7,
          "files": {
            "UXF^table": {
              "name": "Files",
              "comment": "Table comment",
              "fields": {
                "Kind": null,
                "Filename": null
              },
              "records": [
                [
                  "current",
                  "/home/mark/app/rs/tlm/PlaylistsTest.tlm"
                ],
                [
                  "recent1",
                  "/home/mark/app/rs/tlm/PlaylistsTest.tlm"
                ],
                [
                  "recent2",
                  "/home/mark/data/playlists-all.tlm"
                ]
              ]
            }
          }
        }
      }
    },
    "Window": {
      "x": 383,
      "y": 124,
      "width": 590,
      "height": 536,
      "scale": 1.1
    },
    "Magic": {
      "UXF^bytes": "1F8B"
    },
    "Nested Map": {
      "Classical": [
        5,
        true
      ],
      "Modern Instrumental": [
        4,
        false
      ],
      "New Acquistions": [
        1,
        false
      ],
      "Nested List of Lists": {
        "UXF^list": {
          "comment": "List comment",
          "vtype": null,
          "list": [
            [
              5,
              "Classical & <Special>",
              true
            ],
            [
              4,
              "Modern Instrumental",
              false
            ],
            [
              1,
              "New Acquistions",
              null
            ]
          ]
        }
      }
    }
  }
}